package.json

```
@@ -56,7 +56,7 @@
   },
   "dependencies": {
     "@payfunc/model-base": "0.1.34",
-    "@payfunc/model-card": "0.2.40",
+    "@payfunc/model-card": "0.2.42",
```

There is a newer version
```suggestion
    "@payfunc/model-card": "0.2.43",
```